### ISSUE TITLE
resolveColumns() does not take into consideration table's prefix 

### ISSUE THREAD

**Issue by KarinaRashchynskaya** [Soyhuce/next-ide-helper#105]
Hello,

I used the command `php artisan next-ide-helper:models`,  which I expected to resolve my models' attributes from the database.  However, I noticed that no docblock (related to model's attributes) was added to my models' classes after command's execution.
After a research, I realized that the problem begins form the following function :
```
class ResolveModelAttributes implements ModelResolver
{
   // ...
  private function resolveColumns(Model $model): array
      {
          $table = $model->instance()->getTable();
          return $model->instance()
              ->getConnection()
              ->getDoctrineSchemaManager()
              ->listTableColumns($table);
      }
}
```
More specifically, I noticed that when the function is called, a null result is returned. Actually, the issue occurs from the fact that the variable `$table` does not include the table's prefix which exists in `getConnection()` details. As a result, the columns can not be retrieved.
Therefore, my question is, if it's possible to make a change like the following , so in case of a predefined connection name, the method will take the account of it
```
class ResolveModelAttributes implements ModelResolver
{
   // ...
  private function resolveColumns(Model $model): array
      {
          $table = $model->instance()->getConnection()->getTablePrefix().$model->instance()->getTable();
          return $model->instance()
              ->getConnection()
              ->getDoctrineSchemaManager()
              ->listTableColumns($table);
      }
}
```
Thanks a lot!

**Comment by bastien-phi** [comment]
I was not even aware that was possible. If you want to contribute with a PR for that, be my guest !

**Comment by KarinaRashchynskaya** [comment]
> > I was not even aware that was possible. If you want to contribute with a PR for that, be my guest !
> 
> #106

**Comment by bastien-phi** [comment]
Fixed in 0.17.1
